### PR TITLE
xarray tutorial

### PR DESCRIPTION
**Pull request by dcherian**
I may do some small updates tomorrow. But I want to test this out on the binder.

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -32,7 +33,7 @@ dependencies:
   - netcdf4
   - numpy
   - palettable
-  - pandas
+  - pandas<1.1
```
Comment (dcherian) [inline environment.yml:36]: work around https://github.com/pydata/xarray/issues/4283

**Comment by dcherian** [comment]
For the `hvplot` bit the `@pyviz/jupyterlab_pyviz` extension will be necessary. I assume you'll want it for Jim Bednar's bit anyway...

File: environment.yml (hunk 1 above)
Comment (ocefpaf) [reply environment.yml:36]: This won't affect the hub at the moment but I'll propagate the change there tomorrow.

Do you want us to merge this or are you firing up binder from your fork?

Comment (ocefpaf) [reply environment.yml:36]: This won't affect the hub at the moment but I'll propagate the change there tomorrow.

Do you want us to merge this or are you firing up binder from your fork?

Comment (dcherian) [reply environment.yml:36]: > or are you firing up binder from your fork?

Don;t know how to do this!

Let's merge for now.

Comment (ocefpaf) [reply environment.yml:36]: > Don't know how to do this!

You can create your URL with:

`https://binder.pangeo.io/v2/gh/<yourusername>/ohw20-tutorials/<your-branch>?urlpath=lab`

File: environment.yml [hunk 2]
```diff
@@ -9,6 +9,7 @@ dependencies:
   - cartopy
   - cdsapi
   - cf-units
+  - cf_xarray
```
Comment (ocefpaf) [inline environment.yml:12]: YES!!! :tada:

**Pull request merged**